src/renderer/src/index.css

```
@@ -121,6 +121,12 @@
     body {
         @apply bg-background text-foreground;
     }
+
+    html.transparent,
+    html.transparent body {
+        background: transparent !important;
+        background-color: transparent !important;
```

Stylelint (17.2.0)

[error] Unexpected longhand property "background-color" after shorthand property "background" with same value (declaration-block-no-redundant-longhand-properties)

(declaration-block-no-redundant-longhand-properties)